doc/sphinx/source/recipes/recipe_smpi.rst

```diff
@@ -28,9 +29,9 @@ User settings
 
    *Required settings for script*
 
-   * plot_type: cycle (time), zonal (plev, lat), latlon (lat, lon), cycle_latlon (time, lat, lon), cycle_zonal (time, plev, lat)
-   * time_avg: type of time average (monthlyclim, seasonalclim, annualclim)
-   * region: selected region (global, trop, nhext, shext, nhtrop, shtrop, nh, sh, nhmidlat, shmidlat, nhpolar, shpolar, eq)
+   * plot_type: only "cycle_latlon (time, lat, lon)" and "cycle_zonal (time, plev, lat)" available for SMPI; usage is defined in the recipe and is dependent on the used variable (2D variable: cycle_latlon, 3D variable: cycle_zonal)
+   * time_avg: type of time average (only "annualclim" allowed for SMPI, any other settings are not supported for this diagnostic)
```

Comment:
the recipe uses `"yearly"` instead of `"annualclim"`

Reply:
time_avg in recipe_smpi.rst has been corrected (now "yearly").